apps/web/src/views/Swap/V3Swap/index.tsx

```diff
@@ -37,6 +39,11 @@ export function V3SwapForm() {
   const price = useMemo(() => trade && SmartRouter.getExecutionPrice(trade), [trade])
 
   const insufficientFundCurrency = useCheckInsufficientError(trade)
+  const allowedSlippage = useMemo(() => basisPointsToPercent(10), [10])
```

Comment:
Pls check the `useSwapCallArguments` hook, that's where we should integrate with universal-router I believe. No need for slippage and deadline here